packages/cli/src/constants.ts

```diff
@@ -94,3 +94,9 @@ export const CREDENTIAL_BLANKING_VALUE = '__n8n_BLANK_VALUE_e5362baf-c777-4d57-a
 
 export const UM_FIX_INSTRUCTION =
 	'Please fix the database by running ./packages/cli/bin/n8n user-management:reset';
+
+export const TIME = {
```

Comment:
That naming is problematic, imho - it should at least mention that it is TIME_IN_MS or sth.

Reply:
Thank you, please see [9a25301](https://github.com/n8n-io/n8n/pull/7092/commits/9a25301ee96535d1eed8b5223bc5f0af56e7e3dd)